internal/database/dbutil/type_mapping.go

```diff
@@ -0,0 +1,301 @@
+package dbutil
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/oom-ai/oomstore/pkg/errdefs"
+	"github.com/oom-ai/oomstore/pkg/oomstore/types"
+)
+
+func DBValueType(backend types.BackendType, valueType types.ValueType) (string, error) {
+	var mp map[types.ValueType]string
+	switch backend {
+	case types.POSTGRES:
+		mp = valueTypeToPostgresType
+	case types.SQLite:
+		mp = valueTypeToSQLiteType
+	case types.MYSQL:
+		mp = valueTypeToMySQLType
+	case types.CASSANDRA:
+		mp = valueTypeToCassandraType
+	case types.SNOWFLAKE:
+		mp = valueTypeToSnowFlakeType
+	case types.DYNAMODB:
+		mp = valueTypeToDynamoDBType
+	case types.REDSHIFT:
+		mp = valueTypeToRedshiftType
+	default:
+		return "", errdefs.InvalidAttribute(fmt.Errorf("unsupported backend: %s", backend))
+	}
+
+	t, ok := mp[valueType]
+	if !ok {
+		return "", errdefs.InvalidAttribute(fmt.Errorf("unsupported value type: %s", valueType))
+	}
+	return t, nil
+}
+
+func ValueType(backend types.BackendType, dbValueType string) (types.ValueType, error) {
+	var mp map[string]types.ValueType
+	switch backend {
+	case types.POSTGRES:
+		mp = postgresTypeToValueType
+	case types.MYSQL:
+		mp = mysqlTypeToValueType
+	case types.SNOWFLAKE:
+		mp = snowflakeTypeToValueType
+	case types.BIGQUERY:
+		mp = bigQueryTypeToValueType
+	case types.SQLite:
+		mp = sqliteTypeToValueType
+	case types.REDSHIFT:
+		mp = redshiftTypeToValueType
```

Comment:
missing `types.CASSANDRA`?

Reply:
Cassandra is not an offline store, so it doesn't belongs to here

Reply:
but the file not in `database/metadata/dbutil` or `database/online/dbutil`. maybe add a comment to explain why not `types.CASSANDRA`?

Reply:
add a comment